kernel/config.txt

```diff
@@ -21,12 +21,113 @@ necessary. Information about pre-built hardware as a whole will not be covered.
 [0.0] Index
 ________________________________________________________________________________
 
-- Busybox compatibility                                                    [1.0]
-- Never lose your .config ever again                                       [2.0]
-- Removing the perl requirement                                            [3.0]
+- Getting a config                                                         [1.0]
+- Modifying a config                                                       [2.0]
+- Busybox compatibility                                                    [3.0]
+- Never lose your .config ever again                                       [4.0]
+- Removing the perl requirement                                            [5.0]
 
 
-[1.0] Busybox compatibility
+[1.0] Getting a config
+________________________________________________________________________________
+
+The first step in this process is creating a .config file in the Linux source
+tree. It is essentially just a text file containing KEY=value pairs (along with
+comments), each of which control something in the linux build process. Some of
+them are features to be included, some values to be hardcoded.
+
+An important subset of them are the options controlling "drivers", or sections
+of code that give the kernel the capability to interact with various filesystem,
+protocols and hardware components. Drivers configs generally have three possible
+answers: "n" to not compile the driver, "y" to compile it into the kernel
+binary, and "m" to compile them as modules (.ko files stored in
+/usr/lib/modules), that can be loaded and unloaded dynamically by the kernel as
+needed.
```

Comment:
An important subset of the options control "drivers"; sections of code that give the kernel the capability to interact with filesystems, protocols and hardware components. These options typically  have three possible values: "n" to not compile the driver, "y" to compile it into the kernel binary and "m" to compile the driver as a module (.ko files stored in /usr/lib/modules). Modules can be loaded/unloaded dynamically by the kernel as needed.